### PR TITLE
simple cli: provider helper helpers for non napari usage

### PR DESCRIPTION
**Pull request by joshmoore**
cc: @jburel @will-moore 

```
./ome_zarr /tmp/my.zarr
```

### CODE REVIEW

**Comment by will-moore** [comment]
Looks good:

```
$ ./ome-zarr-py/ome_zarr 6001240.zarr/
root_attrs {'multiscales': [{'datasets': [{'path': '0'}, {'path': '1'}], 'version': '0.1'}]}
resolutions ['0', '1']
resolution 0 shape (t, c, z, y, x) (1, 2, 236, 275, 271) chunks ['1', '1', '1', '275', '271'] dtype >u2
resolution 1 shape (t, c, z, y, x) (1, 2, 236, 137, 135) chunks ['1', '1', '1', '137', '135'] dtype >u2
[([dask.array<from-zarr, shape=(1, 2, 236, 275, 271), dtype=>u2, chunksize=(1, 1, 1, 275, 271), chunktype=numpy.ndarray>, dask.array<from-zarr, shape=(1, 2, 236, 137, 135), dtype=>u2, chunksize=(1, 1, 1, 137, 135), chunktype=numpy.ndarray>], {'channel_axis': 1, 'colormap': [<vispy.color.colormap.Colormap object at 0x119709d50>, <vispy.color.colormap.Colormap object at 0x119709d10>], 'contrast_limits': [[0, 1500], [0, 1500]], 'name': ['LaminB1', 'Dapi'], 'visible': [True, True]})]
```

I wonder if you should not ```return``` nothing if not an ome zarr, since you could still get something useful from a non-ome zarr:

```
$ ./ome-zarr-py/ome_zarr 52_plain.zarr/
not an ome-zarr: {zarr}
[(dask.array<from-zarr, shape=(3, 50, 512, 512), dtype=int16, chunksize=(1, 13, 128, 256), chunktype=numpy.ndarray>, {'channel_axis': 1})]
```

File: ome_zarr.py [hunk 1]
```diff
@@ -169,3 +171,16 @@ def get_json(self, subpath):
...
             return {}
+
+
+def info(path):
+    """
+    print information about the ome-zarr fileset
+    """
+    zarr = parse_url(path)
+    if not zarr.is_ome_zarr():
+        print("not an ome-zarr: {zarr}")
```
Comment (will-moore) [inline ome_zarr.py:182]: Was that line meant to be formatted?

**Comment by will-moore** [comment]
Works remotely too:
```
$ ./ome-zarr-py/ome_zarr 'https://s3.embassy.ebi.ac.uk/idr/zarr/v0.1/6001240.zarr/'
root_attrs {'multiscales': [{'datasets': [{'path': '0'}, {'path': '1'}], 'version': '0.1'}]}
resolutions ['0', '1']
resolution 0 shape (t, c, z, y, x) (1, 2, 236, 275, 271) chunks ['1', '1', '1', '275', '271'] dtype >u2
resolution 1 shape (t, c, z, y, x) (1, 2, 236, 137, 135) chunks ['1', '1', '1', '137', '135'] dtype >u2
[([dask.array<from-zarr, shape=(1, 2, 236, 275, 271), dtype=>u2, chunksize=(1, 1, 1, 275, 271), chunktype=numpy.ndarray>, dask.array<from-zarr, shape=(1, 2, 236, 137, 135), dtype=>u2, chunksize=(1, 1, 1, 137, 135), chunktype=numpy.ndarray>], {'channel_axis': 1, 'colormap': [<vispy.color.colormap.Colormap object at 0x11afb4c10>, <vispy.color.colormap.Colormap object at 0x11afb4c50>], 'contrast_limits': [[0, 1500], [0, 1500]], 'name': ['LaminB1', 'Dapi'], 'visible': [True, True]})]
```

File: ome_zarr.py (hunk 1 above)
Comment (joshmoore) [reply ome_zarr.py:182]: Oops.

Comment (joshmoore) [inline ome_zarr.py:182]:
```suggestion
        print(f"not an ome-zarr: {zarr}")
```

**Comment by will-moore** [comment]
I guess for the "don't return nothing" comment above, if you really pass something invalid it doesn't look so good
```
$ ./ome-zarr-py/ome_zarr foo
not an ome-zarr: {zarr}
Traceback (most recent call last):
  File "./ome-zarr-py/ome_zarr", line 10, in <module>
    info(args.path)
  File "/Users/wmoore/Desktop/ZARR/ome-zarr-py/ome_zarr.py", line 184, in info
    reader = zarr.get_reader_function()
  File "/Users/wmoore/Desktop/ZARR/ome-zarr-py/ome_zarr.py", line 87, in get_reader_function
    raise Exception("not a zarr")
Exception: not a zarr
```

**Comment by will-moore** [comment]
You *could* maybe handle a non-ome zarr and give info, but also fine to merge as is and leave that option for later.

File: ome_zarr.py (hunk 1 above)
Comment (will-moore) [reply ome_zarr.py]: I think you want ```print(f"not an ome-zarr: {path}")``` ?

**Comment by joshmoore** [comment]
I don't see the exception on `foo`. Probably will need to come back to that. But :+1: for coming back to non-ome zarrs when we're more sure what we want to do there. Thanks, @will-moore

File: ome_zarr.py (hunk 1 above)
Comment (joshmoore) [reply ome_zarr.py]: I was thinking that thing would eventually have a nice `__str__` to explain _why_ it wasn't an ome-zarr.

**Pull request merged**